### DISCUSSION TITLE
How do I get noice cmdline completions to autoshow while typing instead of pressing tab

### DISCUSSION THREAD

**Discussion by neonvoidx** [saghen/blink.cmp#589] [Q&A]
Not sure if this a question for blink or for noice
![image](https://github.com/user-attachments/assets/fb9f3d4d-ab2e-4335-b529-8930e3136d40)
 Want the above without pressing tab

**Comment by dsully** [comment]
I'm wondering the same thing.

**Comment by saghen** [comment]
This is a question for noice. blink.cmp's cmdline completions already do this
